cli/ioctl/cmd/xrc20/xrc20.go

```diff
@@ -0,0 +1,289 @@
+// Copyright (c) 2019 IoTeX
```

Comment:
This command could be placed in https://github.com/iotexproject/iotex-core/blob/master/cli/ioctl/cmd/action/action.go
then, you can refer to this command as action.Xrc20Cmd